docs/content/connectors/db2-cdc.md

```diff
@@ -23,15 +23,15 @@ using a build automation tool (such as Maven or SBT) and SQL Client with SQL JAR
   <groupId>com.ververica</groupId>
   <artifactId>flink-connector-db2-cdc</artifactId>
   <!-- The dependency is available only for stable releases, SNAPSHOT dependency need build by yourself. -->
-  <version>2.4-SNAPSHOT</version>
+  <version>2.5-SNAPSHOT</version>
 </dependency>
 ```
 
 ### SQL Client JAR
 
 ```Download link is available only for stable releases.```
 
-Download [flink-sql-connector-db2-cdc-2.4-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-db2-cdc/2.4-SNAPSHOT/flink-sql-connector-db2-cdc-2.4-SNAPSHOT.jar) and 
+Download [flink-sql-connector-db2-cdc-2.5-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-db2-cdc/2.5-SNAPSHOT/flink-sql-connector-db2-cdc-2.5-SNAPSHOT.jar) and 
```

Comment:
TODO: we need to update the version to release version `2.4.0` of docs in branch `release-2.4` after release 2.4.0